src/Symfony/Component/Runtime/composer.json

```diff
@@ -20,9 +20,10 @@
         "composer-plugin-api": "^1.0|^2.0"
     },
     "require-dev": {
-        "composer/composer": "^1.0.2|^2.0",
+        "composer/composer": "^2.6",
         "symfony/console": "^6.4|^7.0",
         "symfony/dotenv": "^6.4|^7.0",
+        "symfony/finder": "^6.4",
```

Comment:
This is necessary because [composer/class-map-generator](https://packagist.org/packages/composer/class-map-generator#dev-main) doesn't support `Symfony ^7.0` yet. The highest dep for the Finder must be set to `^6.4` because of this for now.

Reply:
please send a PR to class-map-generator instead

Reply:
It has been dealt by @stloyd [here](https://github.com/composer/class-map-generator/pull/4), thanks!

I'll revert the change once it's merged.

Reply:
Unlocked!

Reply:
I was going to update when I saw you updated the file. Thank you! :slightly_smiling_face: